crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -255,16 +273,20 @@ fn apply_pbr_lighting(
     let clearcoat_R = reflect(-in.V, clearcoat_N);
 #endif  // STANDARD_MATERIAL_CLEARCOAT
 
-    // Diffuse strength is inversely related to metallicity, specular and diffuse transmission
-    let diffuse_color = output_color.rgb * (1.0 - metallic) * (1.0 - specular_transmission) * (1.0 - diffuse_transmission);
+    let diffuse_color = calculate_diffuse_color(
+        output_color.rgb,
+        metallic,
+        specular_transmission,
+        diffuse_transmission
+    );
 
     // Diffuse transmissive strength is inversely related to metallicity and specular transmission, but directly related to diffuse transmission
     let diffuse_transmissive_color = output_color.rgb * (1.0 - metallic) * (1.0 - specular_transmission) * diffuse_transmission;
 
     // Calculate the world position of the second Lambertian lobe used for diffuse transmission, by subtracting material thickness
     let diffuse_transmissive_lobe_world_position = in.world_position - vec4<f32>(in.world_normal, 0.0) * thickness;
 
-    let F0 = lighting::F0(in.material.reflectance, metallic, output_color.rgb);
```

Comment:
is `lighting::F0` still used anywhere after this change?

Reply:
Removed the function.